pkg/pillar/cmd/domainmgr/rkt.go

```diff
@@ -261,6 +258,36 @@ func rktGetHashes() (map[string]string, error) {
 	return m, nil
 }
 
+func getAppManifest(imageHash string) (RktManifest, error) {
```

Comment:
Please rename this function to getRktManifest @adarsh-zededa

Reply:
Done.